client-js/app/elements/labrad-plot.ts

```diff
@@ -1,9 +1,17 @@
 import 'd3';
-
+import * as three from 'three';
 import {viridisData} from '../scripts/colormaps';
 import * as datavault from "../scripts/datavault";
 
+/**
+ * Import the THREE namespace from the module.
+ */
+const THREE = three.default;
```

Comment:
The problem here is that the typescript definition files for these libraries (and most libraries actually) were set up assuming they would be imported using CommonJS style imports like in node, rather than es6 imports. To fix this, you have to change the typings file slightly, like so: https://github.com/labrad/scalabrad-web/commit/7096169119bc2ad8ff4c949d9adeda028a0f0a6e